ImplementationGuide/markdown/Datenobjekte/ISiKTerminKontaktMitGesundheitseinrichtung.md

```diff
@@ -0,0 +1,41 @@
+## ISiKTerminKontaktMitGesundheitseinrichtung [(Encounter)](http://hl7.org/fhir/encounter.html)
+
+---
+
+### Motivation
+
+Das Datenobjekt ISiKKontaktMitGesundheitseinrichtung dient der Verknüpfung des ISiK-Basis-Encounters (ISiKKontaktGesundheitseinrichtung) mit einem Termin (Appointment) und - darauf aufbauend - der Dokumentenkommunikation.
+
+Die Anforderung dieser Verknüpfung stammt aus dem Szenario der Dokument-Übertragung zwischen Patientenportal und krankenhaus-internem Primärsystem (KIS): Dokumente liegen bei Termin-Buchung erst im Patientenportal (im Appointment) vor und werden erst mit Anlage des Encounters in das KIS (etc.) übermittelt. Dazu muss das Appointment mit dem neu angelegten Encounter verknüpft werden, um die Dokumente aus dem Patientenportal darüber abzufragen/zuzuordnen.
```

Comment:
ich würde am Setzende nur "zuzuordnen" schreiben, da Dokumente in dem im Call benannten UseCase ja eigentlich nur aus Richtung Appointment zum Encounter fließen sollen. Bei der umgekehrten Richtung hätte ich nach wie vor Vorbehalte, aber mit der Formulierung "zuornden" bleibt man ja auf der Prosa-Ebene erst einmal vage genug.

Reply:
hier gelöst https://github.com/gematik/spec-ISiK-Terminplanung/pull/87/commits/dc96b3789f31baad87a06bd9bc92d635963e4be4